src/github.com/couchbase/sync_gateway/rest/config.go

```diff
@@ -440,7 +440,7 @@ func setMaxFileDescriptors(maxP *uint64) {
 	actualMax, err := base.SetMaxFileDescriptors(maxFDs)
 	if err != nil {
 		base.Warn("Error setting MaxFileDescriptors to %d: %v", maxFDs, err)
-	} else if maxP != nil {
+	} else if maxP != nil && actualMax > 0 { //Windows no support for setting max file descriptors, returns 0 actualMax
```

Comment:
If someone sets max file descriptors to zero in the config, they aren't going to get any log message about it.  Obviously nobody should be doing that on purpose, but I'm wondering if this makes it harder to debug accidentally setting a zero value.

What do you think about just moving the 'successful' message below into rlimit.go instead?